docs/source/user_guide/faq.rst

```diff
@@ -48,7 +52,7 @@ Should I use `fastplotlib` for making publication figures?
 ----------------------------------------------------------
 
     No, `fastplotlib` is not meant for creating *static* publication figures. There are many other libraries that are well-suited
-    for this task.
+    for this task. However, `fastplotlib` figures can be exported to PNG using ``figure.export()``.
```

Comment:
rephrase, start with "While `fastplotlib` figures can be exported to PNG ..., `fastplotlib` is not intended for creating *static* publication figures..."